operator-framework-core/src/main/java/io/javaoperatorsdk/operator/api/config/ControllerConfigurationOverrider.java

```diff
@@ -89,37 +90,17 @@ private ControllerConfigurationOverrider(ControllerConfiguration<R> original) {
     return this;
   }
 
-  public void replaceDependentResourceConfig(
-      Class<? extends DependentResource<?, R>> dependentResourceClass,
+  public ControllerConfigurationOverrider<R> replacingNamedDependentResourceConfig(String name,
       Object dependentResourceConfig) {
```

Comment:
We don't want at least a marker interface for the dependent resource configurations?

Reply:
Why?

Reply:
Well don't have strong opinion on this, but for now just for forward compatibility?

Reply:
But also fine to have it like this, if you think is right.